### PR TITLE
Upgrade saml bundle

### PR DESCRIPTION
**Pull request by MKodde**
Upgrade the surfnet/stepup-saml-bundle to version ^5.0

See: https://www.pivotaltracker.com/story/show/184071674

### CODE REVIEW

**Review by Shaky212 — COMMENTED** [review]
Looks good, but build fails on xdebug.

**Comment by MKodde** [comment]
This PR needed much TLC in order to pass tests. The one test still failing is the `yarn audit` That will be addressed in #273

File: src/Surfnet/StepupGateway/GatewayBundle/Saml/Proxy/ProxyStateHandler.php [hunk 1]
```diff
@@ -186,11 +186,7 @@ public function getAssertion()
         return $this->get('response_assertion');
     }
 
-    /**
-     * @param $nameId
-     * @return $this
-     */
-    public function saveIdentityNameId($nameId)
+    public function saveIdentityNameId(string $nameId)
```
Comment (Shaky212) [inline src/Surfnet/StepupGateway/GatewayBundle/Saml/Proxy/ProxyStateHandler.php:189]: return type : self ?

**Review by Shaky212 — APPROVED** [review]
Looks good! Just a minor return type thingy.

**Pull request merged**